lib/adzerk/client.rb

```diff
@@ -7,7 +7,11 @@ class Client
                 :advertisers, :flights, :creatives, :creative_maps,
                 :publishers, :invitations, :reports, :channel_site_maps,
                 :logins, :geotargetings, :sitezonetargetings, :categories,
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict resolution accidentally got merged in.

Reply:
Updated